### PR TITLE
Jacobmgevans

### PR DESCRIPTION
**Pull request by JacobMGEvans**

### CODE REVIEW

File: src/routes/Contact.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      </p>
+      <p>
+        <$Labels>
+          Message: <textarea name="message" />
+        </$Labels>
+      </p>
+      <p>
+        <$SubmitButton
+          type="submit"
+          onSubmit={(e) => {
```
Comment (tigerabrodi) [inline src/routes/Contact.tsx:37]: Nit: I enjoy typing the full `event`, just makes it more readable IMO. I leave it up to you.

File: src/routes/Contact.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+const $FormContact = styled('form')`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 20px;
+  border: 1px solid #ccc;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
+  background: url(${backgroundLogo}) no-repeat center;
+  background-size: contain;
```
Comment (tigerabrodi) [inline src/routes/Contact.tsx:60]: Can we put the `background-size` in `background` as well or what do you think?

**Review by tigerabrodi — DISMISSED** [review]
Fine for me, just two small points :blue_heart: :partying_face:

File: src/routes/Contact.tsx (hunk 2 above)
Comment (JacobMGEvans) [reply src/routes/Contact.tsx:60]: Yeah, we could.

File: src/routes/Contact.tsx (hunk 1 above)
Comment (JacobMGEvans) [reply src/routes/Contact.tsx:37]: Sure. 👍🏻

**Pull request merged**